### PR TITLE
Flows improvements

### PR DESCRIPTION
**Pull request by Nitwel**
## Description

This PR adds the option for serial and batched execution of flows that are called by another flow. Before it was all parallel causing the api to easily crash if you had an array with a large length.

## Type of Change

- [ ] Bugfix
- [x] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [x] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

### CODE REVIEW

File: app/src/operations/trigger/index.ts [hunk 1]
```diff
@@ -34,6 +34,51 @@ export default defineOperationApp({
...
+							},
+							{
+								text: '$t:operations.trigger.parallel',
+								value: 'parallel',
+							},
+						],
+					},
+				},
+				schema: {
+					default_value: 'serial',
```
Comment (br41nslug) [inline app/src/operations/trigger/index.ts:63]: Should we make `parallel` the default value as that is the current behavior?

**Review by br41nslug — APPROVED** [review]
LGTM, running flows in series can be very useful and works well in my tests 😄 

```
Parallel
[13:20:35.211] INFO: TEST
[13:20:35.211] INFO: TEST
[13:20:35.211] INFO: TEST
[13:20:35.211] INFO: TEST
[13:20:35.211] INFO: TEST
[13:20:35.212] INFO: TEST
[13:20:35.212] INFO: TEST
[13:20:35.212] INFO: TEST
[13:20:35.212] INFO: TEST
[13:20:35.212] INFO: TEST

Batch
[13:21:19.222] INFO: TEST
[13:21:19.222] INFO: TEST
[13:21:19.231] INFO: TEST
[13:21:19.231] INFO: TEST
[13:21:19.237] INFO: TEST
[13:21:19.237] INFO: TEST
[13:21:19.243] INFO: TEST
[13:21:19.243] INFO: TEST
[13:21:19.249] INFO: TEST
[13:21:19.249] INFO: TEST

Serial
[13:22:12.173] INFO: TEST
[13:22:12.182] INFO: TEST
[13:22:12.187] INFO: TEST
[13:22:12.193] INFO: TEST
[13:22:12.198] INFO: TEST
[13:22:12.204] INFO: TEST
[13:22:12.209] INFO: TEST
[13:22:12.214] INFO: TEST
[13:22:12.219] INFO: TEST
[13:22:12.226] INFO: TEST
```

File: app/src/operations/trigger/index.ts (hunk 1 above)
Comment (rijkvanzanten) [reply app/src/operations/trigger/index.ts:63]: Yes 👍🏻

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Can we put the `undefined` -> `null` change in a separate PR? Ty!

**Comment by Nitwel** [comment]
Sure can do, though why? 😄

**Comment by rijkvanzanten** [comment]
@Nitwel Because they're two different changes, and one of them is done and ready to merge, and the other needs some changes :)

**Comment by Nitwel** [comment]
Changes for fixing undefined can be found in https://github.com/directus/directus/pull/17181

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**